database/migrations/2018_08_15_103716_move_photos.php

```diff
@@ -42,10 +52,32 @@ public function up()
 					$photo->takestamp = ($result->takestamp == 0 || $result->takestamp == null) ? null : date('Y-m-d H:i:s', $result->takestamp);
 					$photo->star = $result->star;
 					$photo->thumbUrl = $result->thumbUrl;
-					$photo->album_id = ($result->album == 0) ? null : $result->album; // IN CASE OF DOWNGRADE
+					$thumbUrl2x = explode('.', $result->thumbUrl);
+					if (count($thumbUrl2x) < 2) {
+						$photo->thumb2x = 0;
+					} else {
+						$thumbUrl2x = $thumbUrl2x[0] . '@2x.' . $thumbUrl2x[1];
+						if (!Storage::exists('thumb/' . $thumbUrl2x)) {
+							$photo->thumb2x = 0;
+						} else {
+							$photo->thumb2x = 1;
+						}
+					}
+
+					$photo->album_id = $album;
```

Comment:
Minor, but this would probably be better on line 36 or so where it's calculated.